src/utils/dnd.ts

```diff
@@ -22,3 +23,98 @@ export const updateLabels = (
 
   return updatedLabels;
 };
+
+type UpdatedLabels = {
+  labels: Label[];
+  answeredLabels: AnsweredLabel[];
+};
+
+const findLabelIndex = (labels: AnsweredLabel[], id: string): number =>
+  labels?.findIndex(({ expected }) => expected.id === id) ?? -1;
+
+type MoveIndices = {
+  srcIdx: number;
+  distIdx: number;
+};
+
+type MoveContext = {
+  labels: Label[];
+  answeredLabels: AnsweredLabel[];
+  srcDroppableId: string;
+  distDroppableId: string;
+  srcLabelIndex: number;
+};
+
+export const trackLabelsChanges = ({
```

Comment:
What do you think about this suggestion ?

```suggestion
export const handleMoveLabel = ({
```